qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOffer.groovy

```diff
@@ -40,10 +56,32 @@ class CreateOffer implements CreateOfferInput, CalculatePrice{
                 offerContent.items,
                 offerContent.selectedCustomerAffiliation)
                 .identifier(identifier)
-                .expirationDate(new Date(2030,12,24)) //todo how to determine this?
+                .expirationDate(new Date(2030,12,24))
```

Comment:
when updating an offer actually the expiration date is not required since it has already been set for the offer creation

Reply:
that is correct, well spotted!

Reply:
should we then not reuse the date from the offer itself?
something like `offerContent.expirationDate`